projects/plugins/crm/api/create_transaction.php

```diff
@@ -27,19 +27,37 @@
 		$json_params = file_get_contents( 'php://input' );
 		$new_trans   = json_decode( $json_params, true );
 
-		// REQUIRED
-		$orderid = -1;
+// REQUIRED FIELDS
+$required_fields = array(
+	'orderid',
+	'email',
+);
```

Comment:
I don't understand why we're doing this?

Reply:
To ensure we get the required fields

Reply:
I added `status` and `total` as required fields and removed `email`